ingest/rules/curate.smk

```diff
@@ -57,37 +53,43 @@ rule curate:
         annotations_id=config["curate"]["annotations_id"],
         id_field=config["curate"]["output_id_field"],
         sequence_field=config["curate"]["output_sequence_field"],
+    benchmark:
+        "benchmarks/curate.txt"
+    log:
+        "logs/curate.txt"
     shell:
-        """
-        (cat {input.sequences_ndjson} \
+        r"""
+        exec &> >(tee {log:q})
+
+        cat {input.sequences_ndjson:q} \
             | augur curate rename \
                 --field-map {params.field_map:q} \
             | augur curate normalize-strings \
             | augur curate transform-strain-name \
-                --strain-regex {params.strain_regex} \
-                --backup-fields {params.strain_backup_fields} \
+                --strain-regex {params.strain_regex:q} \
+                --backup-fields {params.strain_backup_fields:q} \
             | augur curate format-dates \
-                --date-fields {params.date_fields} \
-                --expected-date-formats {params.expected_date_formats} \
+                --date-fields {params.date_fields:q} \
+                --expected-date-formats {params.expected_date_formats:q} \
             | augur curate parse-genbank-location \
-                --location-field {params.genbank_location_field} \
+                --location-field {params.genbank_location_field:q} \
             | augur curate titlecase \
-                --titlecase-fields {params.titlecase_fields} \
-                --articles {params.articles} \
-                --abbreviations {params.abbreviations} \
+                --titlecase-fields {params.titlecase_fields:q} \
+                --articles {params.articles:q} \
+                --abbreviations {params.abbreviations:q} \
             | augur curate abbreviate-authors \
-                --authors-field {params.authors_field} \
-                --default-value {params.authors_default_value} \
-                --abbr-authors-field {params.abbr_authors_field} \
+                --authors-field {params.authors_field:q} \
+                --default-value {params.authors_default_value:q} \
+                --abbr-authors-field {params.abbr_authors_field:q} \
             | augur curate apply-geolocation-rules \
-                --geolocation-rules {input.geolocation_rules} \
+                --geolocation-rules {input.all_geolocation_rules:q} \
```

Comment:
```suggestion
                --geolocation-rules {input.geolocation_rules:q} \
```

Reply:
Thanks! Fixed in [force-push](https://github.com/nextstrain/pathogen-repo-guide/compare/8c6c6752a079bf4e69a98759f38c2d2f078866d8..8b3b2644db7ab693cf804b3a91cdfa0901f95b9d).